client/lib/plugins/store.js

```diff
@@ -310,8 +305,7 @@ PluginsStore = {
 };
 
 PluginsStore.dispatchToken = Dispatcher.register( function( payload ) {
```

Comment:
We really don't care about `payload` here I don't think. In this case, we can use destructuring to eliminate some variables and leave more room for the actual code we care about.

```js
Dispatcher.register( function( { action } ) {
	…
} );
```